packages/cli/src/commands/generate.js

```diff
@@ -1,6 +1,15 @@
 export const command = 'generate <type>'
```

Comment:
FYI: somewhere along the way we discussed changing `<type>` here for `yarn rw generate --help`. However, I want to confirm we should leave it as is (just in case that conversation was still lingering on a ToDo list somewhere).